test/demo/index.tsx

```diff
@@ -3,12 +3,12 @@ import { createRoot } from 'react-dom/client'
 
```

Comment:
Can you use `option` in demo instead of `menu`? For instance, in “search” and list hotkey examples.